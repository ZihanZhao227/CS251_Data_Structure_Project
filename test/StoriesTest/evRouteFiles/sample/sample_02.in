6
0 0 0 CITY StartCity
1 50 0 RESTAURANT Restaurant1
2 100 0 CHARGING_STATION ChargingStation1
3 200 0 TOURIST_ATTRACTION Museum1
4 300 0 CHARGING_STATION ChargingStation2
5 400 0 CITY DestinationCity