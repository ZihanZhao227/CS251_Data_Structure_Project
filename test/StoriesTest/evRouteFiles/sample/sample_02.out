0 5 80.0 -1.0
NO_PATH